src/validations/auth.validation.js

```diff
@@ -45,19 +44,55 @@ const signupSchema = Joi.object({
     })
 }).options({ abortEarly: false });
 
-export const signupValidator = (req, res, next) => {
+const loginSchema = Joi.object({
+  email: Joi.string()
+    .email({ minDomainSegments: 2 })
+    .required()
+    .trim()
+    .messages({
+      'string.base': 'Invalid type, email must be a string',
+      'string.empty': 'Please enter email',
+      'string.pattern.base': 'Enter valid email i.e: gustave@gmail.com',
+      'any.required': 'Email is required'
+    }),
+  password: Joi.string()
+    .trim()
+    .min(8)
+    .required()
+    .messages({
+      'string.base': 'Invalid type, password must be a string',
+      'string.empty': 'Please enter password',
+      'string.min': 'Password must be atleast 8 characters long',
+      'any.required': 'Password is required'
+    })
+}).options({ abortEarly: false });
+
+export const validateSignup = (req, res, next) => {
   const userInputValidator = signupSchema.validate(req.body);
```

Comment:
@victor-abz  It's better if u call it validateUserInput to have better meaning

Reply:
Ok